### DISCUSSION TITLE
No WordPress-Core, WordPress-Extra Ruleset?

### DISCUSSION THREAD

**Discussion by lichaar** [PHP-CS-Fixer/PHP-CS-Fixer#6822] [Q&A] [answered]
## Support question
Does PHP CS Fixer supports WordPress-Core, WordPress-Extra Ruleset?
It seems only PHP-CBF offers this which fixes the file based on the ruleset provided for PHP-CS but I couldn't find any similar documentation for PHP-CS-Fixer.

**Comment by SpacePossum** [comment, marked as answer]
Hi and thanks for your question!

Historically the style this tool was based on didn't look close to the one word press uses. Later more and more support for other styles was added. However since word press typically uses different tooling (like you refer to) and with a lack of contributors here from the word press community makes it that we never really added a dedicated word press rule set. Would be great to have it as word press is a big part of PHP, but if not many of its developers can help us out here I don't think we can afford the development and maintenance effort.
